src/client/__tests__/testKits/apis/apiDependenciesKit.ts

```diff
@@ -0,0 +1,16 @@
+import { GitHubApiTestKit } from './githubApi';
+import { IServicesDependencies } from '../../../services/services';
+
+export class ApiDependenciesKit {
```

Comment:
Rename this file to `ApiDependenciesKit.ts`, same as the class name.

Reply:
Done